server/accounts.go

```diff
@@ -2937,7 +2937,8 @@ func (dr *DirAccResolver) Start(s *Server) error {
 		}
 	}
 	const accountPackRequest = "$SYS.ACCOUNT.CLAIMS.PACK"
-	const accountLookupRequest = "$SYS.ACCOUNT.*.CLAIMS.LOOKUP"
+	const accountLookupRequest = "$SYS.REQ.ACCOUNT.*.CLAIMS.LOOKUP"
```

Comment:
Originally this was used only in this function, but now I wonder if we should move then to events.go (where you have defined number of tokens and index of account).
Also, we could make it a subject with the `%s` token, like `const accountLookupRequest = "$SYS.REQ.ACCOUNT.%s.CLAIMS.LOOKUP"`.
Here, to subscribe you would sprintf with this constant and pass "*" for the "%s".

Reply:
will make these changes